### DISCUSSION TITLE
Closing popover on outside click

### DISCUSSION THREAD

**Discussion by khizarshaikh** [Akryum/v-tooltip#673] [Q&A]
Actually i am using  vue2-dropzone inside v-tool tip pop over and i want  to close popover on outside clicks which works fine with auto-hide: true but when i try to select any file using vue2-dropzone then also the popover gets closed which is inside v-tooltip. so how can i prevent it in vue2-dropzone which is placed inside the template block of v-tool tip.

`<v-popover :open="showPopOver" offset="-100%" placement="right-start" :auto-hide="true">
                            <button type="button"
                                    class="btn btn-primary text-nowrap"
                                    v-on:click="togglePopover()"
                                    title="Upload Logo">
                                Upload Logo
                            </button>
                            <template slot="popover">
                                <div class="popover-wrapper shadow ">
                                    <drag-drop-image></drag-drop-image>
                                </div>
                            </template>
                        </v-popover>`

**Comment by ballonura** [comment]
Hey, any way please to close tooltip (trigger: 'click') by clicking outside the tooltip ?

**Comment by nik0145** [comment]
+1 
I have the same problem, with date picker

**Comment by abdulqadir-botsify** [comment]
set the following config
VTooltip.options.themes.tooltip.triggers = ['hover', 'focus', 'touch','click'];
